oeps/oep-0011-bp-FED-technology.rst

```diff
@@ -100,35 +109,28 @@ Technology Selection
    `edX ESLint Config`_ is made available as an npm package that can be
    installed into any Open edX package.
 
-   **Exception**: The `edX ESLint Config for ES5`_ may be used where ES5
+   **Exception**: The `edX ESLint Config for ES5`_ may be used only where ES5
```

Comment:
Nit: Re-order so that #4 (ES Lint) is above #3 (NPM).
Rationale: Then, the packaging/dependency related items (npm, package.lock, and greenkeeper) are co-located.